test/unit/gateways/ebanx_test.rb

```diff
@@ -54,6 +54,17 @@ def test_successful_purchase_with_soft_descriptor
     assert_success response
   end
 
+  def test_successful_purchase_with_merchant_payment_code
```

Comment:
For sanity sake can we also add a unit test that shows the order_id is still successfully mapped to merchant payment code if one is not provided in the options?